app/javascript/components/Button/index.scss

```diff
@@ -31,7 +31,8 @@
   }
 
   // Colors
-  &:enabled {
+  &:enabled,
+  &.Button__link {
```

Comment:
Current button components dont support hrefs, so I've added this class make it work with not only button elements

Reply:
Good point. I think we should update the button component to support links, it's a simple thing I know I've implemented on multiple projects.

Reply:
I've fixed this issue by changing `&:enabled` to `&:not(:disabled)`.
With that change we don't need an additional `Button__link` class to make links work with the button classes.